vispy/geometry/generation.py

```diff
@@ -289,3 +289,56 @@ def create_arrow(rows, cols, radius=0.1, length=1.0,
         faces = np.vstack((faces, md_cyl.get_faces()+nbr_verts_con))
 
     return MeshData(vertices=verts, faces=faces)
+
+
+def create_implicit_mesh(xs, ys, zs):
```

Comment:
`SurfacePlotVisual` has a vectorized method for generating the mesh vertices and indices; you should be able to adapt that for `create_implicit_mesh()`. (Ideally, `SurfacePlotVisual` could be reimplemented as a very lightweight subclass of `ImplicitMeshVisual`)